eventually/tests/unittests/utils/testteamviewsfunctions.py

```diff
@@ -0,0 +1,270 @@
+from django.test import TestCase
+from authentication.models import CustomUser
+from utils import team_views_functions
+
+
+class UtilsTeamViewsFunctionsTestCase(TestCase):
+
+    def setUp(self):
+
+            CustomUser(
+                       id=20,
+                       email='email@email.com',
+                       password='1111').save()
+
+            CustomUser(
+                       id=200,
+                       email='tets@email.com',
+                       password='0000').save()
+
+    def test_find_users_by_id_success(self):
+        members_list = team_views_functions.find_users([20])
```

Comment:
it is good practice to name this kind of data something like `actual_members_list` or `current_members_list`